bin/analysis/get_app_analytics.py

```diff
@@ -0,0 +1,120 @@
+import logging
+import emission.core.get_database as edb
+import emission.storage.timeseries.abstract_timeseries as esta
+import pandas as pd
+import matplotlib.pyplot as plt
+import datetime as dt
+import matplotlib.dates as mdates
+
+
+def get_app_analytics():
+    df = pd.DataFrame()
+    for user in edb.get_uuid_db().find():
+        user_df = esta.TimeSeries.get_time_series(user['uuid']).get_data_df("stats/server_api_time", time_query=None)
+        if not user_df.empty:
+            df = df.append(user_df, ignore_index = True)
```

Comment:
At this point, you have one giant dataframe with data combined across all users, right?

e.g. if you have users u1 and u2, df will consist of 

<u1 row1>
<u1 row2>
<u1 row3>
<u1 row4>
....
<u2 row1>
<u2 row2>
....

so if you plot the calls to metrics/timestamp, you won't see how user1 accessed the dashboard versus user2 accessed the dashboard. Instead, you will get a combined plot across both users, right?